index.html

```diff
@@ -36,7 +43,7 @@
       
     </script>
     <!-- local styles for this document -->
-	<link rel="stylesheet" href="local.css">
+	<link rel="stylesheet" data-import href="local.css">
```

Comment:
For this to work, i think you need to include the following in respecConfig:

````
	postProcess: [
		async function importStyleSheet() { 
			const elems = document.querySelectorAll(`link[rel='stylesheet'][data-import]`)
			await Promise.all(
				[...elems].map(async link => {
					const text = await fetch(link.href).then(r => r.text())
					const style = document.createElement("style")
					style.textContent = text
					link.replaceWith(style)
					})
				)
			}
		],
````

Reply:
fixed